src/sentry/api/endpoints/project_platforms.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+
+from django.utils import timezone
+from rest_framework.response import Response
+from rest_framework import serializers
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.serializers import serialize
+from sentry.models import ProjectPlatform
+
+
+class ProjectPlatformSerializer(serializers.Serializer):
+    project_id = serializers.IntegerField()
+    platform = serializers.CharField()
+
+
+class ProjectPlatformsEndpoint(ProjectEndpoint):
+    def get(self, request, project):
+        queryset = ProjectPlatform.objects.filter(
+            project_id=project.id
+        ).values('project_id', 'platform', 'date_added', 'last_seen', 'date_chosen')
```

Comment:
we use camelCase in API responses, and things like 'dateCreated' (vs date_added) are common